### PR TITLE
[SWTCH-1158] Add terms and condition textbox

### PR DESCRIPTION
**Pull request by davycastillo**

### CODE REVIEW

File: src/app/routes/applications/new-staking-pool/new-staking-pool.component.html [hunk 1]
```diff
@@ -106,13 +106,25 @@ <h4 mat-dialog-title class="mb-0 d-flex justify-content-between align-items-cent
                         </div>
                     </div>
                 </div>
+                <div class="col-12" *appIsFeatureEnabled>
```
Comment (dawidgil) [inline src/app/routes/applications/new-staking-pool/new-staking-pool.component.html:109]: You do not need this directive here. This view is already protected :)

File: src/app/routes/applications/new-staking-pool/new-staking-pool.component.html [hunk 2]
```diff
@@ -106,13 +106,25 @@ <h4 mat-dialog-title class="mb-0 d-flex justify-content-between align-items-cent
...
+                    <div class="d-flex align-items-center">
+                        <mat-label class="ml-3 pl-1">Terms and Condition</mat-label>
+                        <mat-icon class="f-16 cursor-pointer icon-color pt-1 ml-2"
+                            matTooltip="What to add in the Terms and Condition">
+                            help_outline
+                        </mat-icon>
+                    </div>
+                    <mat-form-field appearance="outline">
+                        <textarea class="textarea-staking" matInput placeholder="Terms and Condition" rows="6"
```
Comment (dawidgil) [inline src/app/routes/applications/new-staking-pool/new-staking-pool.component.html:118]: This textarea is for all terms and conditions or for one? If for all then probably we will have a problem with splitting this in staking view into separate lines of a list. If for one then we will need button to add it, and place where we could display them and delete them. Does it make sense?

Comment (davycastillo) [reply src/app/routes/applications/new-staking-pool/new-staking-pool.component.html:118]: Yeah, I will create some options for this.

**Review by dawidgil — APPROVED** [review]
Lgtm

**Pull request merged**